src/app/routes/home/getInitialData/index.test.js

```diff
@@ -96,22 +86,20 @@ describe('Get initial data from front page', () => {
   });
 
   it('should return page data for misconfigured service without radio schedules, but with radio schedules on front page', async () => {
-    getConfig.mockImplementationOnce(() => ({
-      radioSchedule: {
-        hasRadioSchedule: false,
-        onFrontPage: true,
-      },
-    }));
-
     fetchMock.mock('http://localhost/mock-frontpage-path.json', frontPageJson);
     fetchMock.mock(
       'http://localhost/hausa/bbc_hausa_radio/schedule.json',
-      radioScheduleJson,
+      null,
```

Comment:
I can't see why this mock has changed, could this cause this test to pass in error?

Reply:
This test was failing for me. Reading the description didn't match up with the test fixture - `should return page data for misconfigured service without radio schedules` - it was passing in radio schedule data when the description says otherwise. Removing the radio schedule data made the test pass. I assumed this was a mistake by whoever originally wrote the test and that it was passing because of a misconfigured mock.